src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -118,10 +118,8 @@ public static partial class ImmutableArray
         public System.Collections.Immutable.ImmutableArray<T> Add(T item) { throw null; }
         public System.Collections.Immutable.ImmutableArray<T> AddRange(System.Collections.Generic.IEnumerable<T> items) { throw null; }
         public System.Collections.Immutable.ImmutableArray<T> AddRange(System.Collections.Immutable.ImmutableArray<T> items) { throw null; }
-#if !NETSTANDARD1_0
```

Comment:
System.Collections.Immutable is widely used, including by Roslyn.  Just want to re-confirm there are no known remaining important dependencies on the .NET Standard 1.0 build?

Reply:
Let me provide clarity on the overall communication strategy and then go into specifics for Roslyn.

1. The change to remove unsupported application tfms (.NETCoreApp < 3.1 and .NETFramework < 4.6.1) and pre netstandard2.0 library assets (.NETStandard < 2.0) was approved by leadership months ago. 
2. The first wave of this effort (removing harvested assets) was presented to Tactics months ago.
3. In yesterday's meeting I communicated that I'll start with trimming the remaining 25 shipping packages and we agreed on to a) send out a separate mail to Tactics in case people where not present in the meeting b) send out a notice to our first party customers via the breaking change notice alias which includes filing a public breaking change issue.
4. The docs team will then later convert the breaking change issue into a docs page.

In parallel to submitting these PRs (one for .NETStandard, another one for .NETFramework and the last one for .NETCoreApp) I'm filing the breaking change issue to send out the mail to Tactics and the bcn alias. I plan to do that later this week.

In regards to roslyn as a consumer, I found three code pieces where they still consume the netstandard1.x asset of 
`System.Collections.Immutable` and `System.Reflection.Metadata`. All three projects will need to react to this change by either a) upgrading the tfms or if that's not possible/desirable b) continue referencing the 5.0.0 package even after the 6.0.0 shipped (only for configurations that are affected by this change). Thanks for the feedback, I'll make sure to include the roslyn team in the breaking change notice and will proactively reach out to them.

Customers who try to upgrade these packages to the 6.0.0 version (or to the preview 6 pre-release versions) and still target netstandard1.x, < net461 or < netcoreapp3.1 in their applications will hit an upgrade error in the NuGet Package Explorer or when trying to restore the newer version of the packages i.e. via `dotnet restore`. The error will inform customers about which packages don't support the tfms in use anymore.

Please let me know if you have further suggestions regarding the handling and communication of the breaking changes.